docs/index.rst

```diff
@@ -1,50 +1,155 @@
+        position: absolute;
+        right: 0;
+        top: 120px;
+      }
+      div#badges a {
+        display: block;
+        padding: 1pt 0;
+        text-align: right;
+      }
+      div#collage {
+        left: 100pt;
+        position: relative;
+        top: 10pt;
+        transform: matrix(1, 0, -0.29, 1, 0, 0);
+      }
+      div#collage a.box {
+        border-radius: 5px;
+        display: inline-block;
+        font-family: Verdana;
+        position: absolute;
+        text-align: center;
+        user-select: none;
+      }
+      div#collage a.box:hover {
+        box-shadow: 2px 2px 2px #222;
+      }
+      div#collage a.box#box1 {
+        background-color: #4684b6;
+        color: white;
+        height: 108pt;
+        padding: 28pt 0;
+        top: 0;
+        width: 111pt;
+      }
+      a.box#box1 div:first-child { font-size: 22pt; }
+      a.box#box1 div:last-child  { font-size: 16pt; }
+      div#collage a.box#box2 {
+        background-color: #2d659c;
+        color: white;
+        height: 50pt;
+        left: 113pt;
+        padding: 11pt 0;
+        top: 58pt;
+        width: 56pt;
+      }
+      a.box#box2 div:first-child { font-size: 11pt; }
+      a.box#box2 div:last-child  { font-size: 10pt; }
+      div#collage a.box#box3 {
+        background-color: #a2c1da;
+        color: black;
+        height: 82pt;
+        left: 171pt;
+        padding: 10pt 0;
+        top: 58pt;
+        width: 93pt;
+      }
+      a.box#box3 div:first-child { font-size: 36pt; }
+      a.box#box3 div:last-child  { font-size: 10pt; }
+      div#collage a.box#box4 {
+        background-color: #2c5735;
+        color: white;
+        font-size: 12pt;
+        height: 22pt;
+        left: 25pt;
+        padding: 2pt;
+        top: 129pt;
+        transform: rotate(-90deg);
+        width: 60pt;
+      }
+      div#collage a.box#box5 {
+        background-color: #ffde56;
+        color: black;
+        height: 99pt;
+        left: 68pt;
+        padding: 17pt 0;
+        top: 110pt;
+        width: 101pt;
+      }
+      a.box#box5 div:first-child { font-size: 30pt; }
+      a.box#box5 div:last-child  { font-size: 16pt; }
+      div#collage a.box#box6 {
+        background-color: #a2a84a;
+        color: black;
+        font-size: 14pt;
+        height: 27pt;
+        left: 171pt;
+        padding: 3pt;
+        top: 142pt;
+        width: 125pt;
+      }
+    </style>
 
-        <div class="small-links">
-          <div><a class="reference internal" href="start/install.html">
-           &#x25AA; Installation
-          </a></div>
-          <div><a class="reference internal" href="develop/index-develop.html">
-           &#x25AA; Development
-          </a></div>
-          <div><a class="reference internal" href="genindex.html">
-           &#x25AA; Index
-          </a></div>
-        </div>
+    <div id="dtcontainer">
+      <div id="description"><p>
+        <b>Datatable</b> is a python library for manipulating tabular data.
```

Comment:
I understand that this is a first word in the sentence, but still, shouldn't it be `datatable` instead of `Datatable`?

Reply:
Yeah, I tried that at first, but it just doesn't look as good

Reply:
May be we unify how `datatable` word is spelled and looks everywhere on the docs?